gradle.properties

```diff
@@ -2,10 +2,11 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m
 
 deployUrl=file:///Users/cer/.m2/testdeploy
 
-eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,file:///Users/cer/.m2/testdeploy
+eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-release/,file:///Users/cer/.m2/testdeploy,https://dl.bintray.com/eventuateio-oss/eventuate-maven-rc
 
-springBootVersion=1.4.5.RELEASE
-eventuateClientVersion=0.20.1.RELEASE
-eventuateTramVersion=0.7.0.RELEASE
+springBootVersion=2.1.1.RELEASE
+eventuateClientVersion=0.22.0.RC1
```

Comment:
`eventuateClientVersion` is unnecessary, right?

Reply:
I will check